### PR TITLE
Add clusteroperators and clusterversions to explain tests

### PR DESCRIPTION
**Pull request by soltysh**
This is a followup from https://github.com/openshift/origin/pull/22696 now that https://github.com/openshift/cluster-version-operator/pull/168 merged 
/assign @mfojtik

### CODE REVIEW

File: test/extended/cli/explain.go [hunk 1]
```diff
@@ -87,12 +86,12 @@ var (
...
-		// schema.GroupVersionResource{Group: "machineconfiguration.openshift.io", Version: "v1", Resource: "controllerconfigs"},
-		// schema.GroupVersionResource{Group: "machineconfiguration.openshift.io", Version: "v1", Resource: "kubeletconfigs"},
-		// schema.GroupVersionResource{Group: "machineconfiguration.openshift.io", Version: "v1", Resource: "machineconfigpools"},
-		// schema.GroupVersionResource{Group: "machineconfiguration.openshift.io", Version: "v1", Resource: "machineconfigs"},
-		// schema.GroupVersionResource{Group: "machineconfiguration.openshift.io", Version: "v1", Resource: "mcoconfigs"},
+		// {Group: "machineconfiguration.openshift.io", Version: "v1", Resource: "containerruntimeconfigs"},
+		// {Group: "machineconfiguration.openshift.io", Version: "v1", Resource: "controllerconfigs"},
+		// {Group: "machineconfiguration.openshift.io", Version: "v1", Resource: "kubeletconfigs"},
+		// {Group: "machineconfiguration.openshift.io", Version: "v1", Resource: "machineconfigpools"},
+		// {Group: "machineconfiguration.openshift.io", Version: "v1", Resource: "machineconfigs"},
```
Comment (runcom) [inline test/extended/cli/explain.go]: I'm not sure how we can move this type to openshift/api. machineconfigs depends on Ignition which we don't control so we can't add tags for autogenerated stuff. We're moving what we can from above in https://github.com/openshift/api/pull/301 but there's something which we need to understand and fix yet

**Comment by ingvagabund** [comment]
Is
```
$ oc explain clusteroperators
KIND:     ClusterOperator
VERSION:  config.openshift.io/v1

DESCRIPTION:

FIELDS:
   apiVersion	<string>
     APIVersion defines the versioned schema of this representation of an
     object. Servers should convert recognized schemas to the latest internal
     value, and may reject unrecognized values. More info:
     https://git.k8s.io/community/contributors/devel/api-conventions.md#resources

   kind	<string>
     Kind is a string value representing the REST resource this object
     represents. Servers may infer this from the endpoint the client submits
     requests to. Cannot be updated. In CamelCase. More info:
     https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds

   metadata	<Object>
     Standard object's metadata. More info:
     https://git.k8s.io/community/contributors/devel/api-conventions.md#metadata

   spec	<map[string]> -required-
     spec hold the intent of how this operator should behave.

   status	<Object>
     status holds the information about the state of an operator. It is
     consistent with status information across the kube ecosystem.
```
the expected output? No need to describe `spec`/`status` fields?

File: test/extended/cli/explain.go (hunk 1 above)
Comment (soltysh) [reply test/extended/cli/explain.go]: @runcom not everything needs to be in openshift/api to get it tested. The gist is to publish those information when creating those CRDs.

**Comment by soltysh** [comment]
@ingvagabund - yes, thanks!

**Pull request merged**